lib/unzip.js

```diff
@@ -23,10 +23,21 @@ module.exports = function (input, isGZip, isSprite, callback) {
     }
     return JSZip.loadAsync(input)
         .then(function (zip) {
-            return zip.file(isSprite ? 'sprite.json' : 'project.json').async('string')
-                .then(function (project) {
-                    return callback(null, [project, zip]);
-                });
+            let file;
+            if (isSprite) {
+                file = zip.file('sprite.json');
```

Comment:
I think we want to treat sprite files the same way we treat project files. I just confirmed that this was something that worked for .sprite2 files in the 2.0 editor.